output/runme.cs

```diff
@@ -2,8 +2,8 @@
  public class runme {
      static void Main() {
          try {
-             Console.WriteLine(secrethub.Read("marton/demo/username"));
-             Console.WriteLine(secrethub.Read("marton/asfdasjfk/asdfasdf"));
+             secrethub.Write("horiaculea/flags/infusedtea", "colazero");
+             Console.WriteLine(secrethub.Exists("marton/asfdasjfk/asdfasdf"));
```

Comment:
```suggestion
             Console.WriteLine(secrethub.Read("marton/demo/username"));
             Console.WriteLine(secrethub.Read("marton/asfdasjfk/asdfasdf"));
```

Tiny thing: let's revert this because I changed this in a separate PR to use secrets that we all have access to.